docs/content/latest/deploy/docker/docker-swarm.md

```diff
@@ -167,7 +167,8 @@ $ docker service create \
 --publish 7000:7000 \
 yugabytedb/yugabyte:latest /home/yugabyte/bin/yb-master \
 --fs_data_dirs=/mnt/data0 \
---master_addresses=yb-master1:7100,yb-master2:7100,yb-master3:7100 \
+--master_addresses=tasks.yb-master1:7100,tasks.yb-master2:7100,tasks.yb-master3:7100 \
+--rpc_bind_addresses=0.0.0.0:7100 \
```

Comment:
rpc_bind_addresses should be set to the node's IP not 0.0.00. For master1 that should be`tasks.yb-master1:7100`

Reply:
okay. I'll make this change.